pages/apis/webhooks.md

```diff
@@ -14,16 +14,17 @@ You can subscribe to one or more of the following events:
   <tr><th>Event</th><th>Description</th></tr>
 </thead>
 <tbody>
-  <tr><th>ping</th><td>Webhook notification settings have changed</td></tr>
-  <tr><th>build.scheduled</th><td>A build has been scheduled</td></tr>
-  <tr><th>build.running</th><td>A build has started running</td></tr>
-  <tr><th>build.failing</th><td>A build is failing</td></tr>
-  <tr><th>build.finished</th><td>A build has finished</td></tr>
-  <tr><th>job.scheduled</th><td>A job has been scheduled</td></tr>
-  <tr><th>job.started</th><td>A command step job has started running on an agent</td></tr>
-  <tr><th>job.finished</th><td>A job has finished</td></tr>
-  <tr><th>job.activated</th><td>A block step job has been unblocked using the web or API</td></tr>
+  <tr><th><code>ping</code></th><td>Webhook notification settings have changed</td></tr>
+  <tr><th><code>build.scheduled</code></th><td>A build has been scheduled</td></tr>
+  <tr><th><code>build.running</code></th><td>A build has started running</td></tr>
+  <tr><th><code>build.failing</code></th><td>A build is failing</td></tr>
+  <tr><th><code>build.finished</code></th><td>A build has finished</td></tr>
+  <tr><th><code>job.scheduled</code></th><td>A job has been scheduled</td></tr>
+  <tr><th><code>job.started</code></th><td>A command step job has started running on an agent</td></tr>
+  <tr><th><code>job.finished</code></th><td>A job has finished</td></tr>
+  <tr><th><code>job.activated</code></th><td>A block step job has been unblocked using the web or API</td></tr>
```

Comment:
Fixing this:
![CleanShot 2023-06-30 at 15 09 32](https://github.com/buildkite/docs/assets/11305534/d7dc1d69-a6a7-4e16-b142-fc3fea82fdf5)